app/lib/common/widgets/acter_icon_picker/picker/acter_icon_picker.dart

```diff
@@ -0,0 +1,190 @@
+import 'package:acter/common/toolkit/buttons/primary_action_button.dart';
+import 'package:acter/common/widgets/acter_icon_picker/model/color_data.dart';
+import 'package:acter/common/widgets/acter_icon_picker/model/acter_icons.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+
+void showActerIconPicker({
+  required BuildContext context,
+  required final Color selectedColor,
+  required final ActerIcons selectedIcon,
+  final Function(Color, ActerIcons)? onIconSelection,
+}) {
+  showModalBottomSheet(
+    showDragHandle: true,
+    useSafeArea: true,
+    context: context,
+    isDismissible: true,
+    isScrollControlled: true,
+    builder: (context) {
+      return ActerIconPicker(
+        selectedColor: selectedColor,
+        selectedIcon: selectedIcon,
+        onIconSelection: onIconSelection,
+      );
+    },
+  );
+}
+
+class ActerIconPicker extends StatefulWidget {
```

Comment:
This class is for ActerIcon Picker.
- It is manage as independent Component which can be reused based on the requirement.
- For shake of simplicity, Entire picker states are managed by basic `ValueNotifier` entire state management kept within class and avoided providers here to make it more independent widget.

![Screenshot 2024-09-05 at 1 56 03 PM](https://github.com/user-attachments/assets/62805148-b601-4cae-98cb-813272fee7b3)